src/vite/vuetify-styles-plugin.ts

```diff
@@ -80,6 +80,9 @@ export function vuetifyStylesPlugin(
         const file = /^\/@fs\/plugin-vuetify\/lib\/(.*?)(\?.*)?$/.exec(id)![1]
         return tempFiles.get(file)
       }
+
+      if (id.includes('plugin-vuetify/lib'))
+        return ''
```

Comment:
Here the solution

Reply:
requires patch release or only for test?

Reply:
forgot it